src/Controller/AddOnOptionsController.py

```diff
@@ -362,7 +362,7 @@ def accepting(self):
             stream.close()
         # Close the Add-On Options Window after saving
 
-        if self.window.has_rtss:
+        if hasattr(self.window, 'structures_tab'):
```

Comment:
is it an error if it doesn't have the structures_tab ?
if so, is it a big problem, or just something to be noted, perhaps in a log file.
See https://docs.python.org/3/howto/logging.html

Reply:
Currently, the structures_tab is only created when an RTSTRUCT is present. It's not an error if there's no structures tab. If an RTSTRUCT is created during runtime, then a structures tab will also be created.